docs/encyclopedia/detecting-activity-failures.mdx

```diff
@@ -206,6 +206,10 @@ Throttling is implemented as follows:
   - Sends the most recent Heartbeat.
   - Sets the timer again.
 
+Throttling allows the Worker to reduce network traffic and load on the Temporal Service by suppressing Heartbeats that aren’t necessary to prevent a Heartbeat Timeout.
+While this may result in some messages not being delivered to the Temporal Service, throttling does not apply to the final Heartbeat message in the case of Activity Failure.
```

Comment:
I think we have to be really careful about clarifying what we mean when we say that messages sometimes aren't delivered -- see https://docs.temporal.io/cloud/service-availability#throughput.

Reply:
Good point. Removed that part.